ironfish/src/consensus/verifier.ts

```diff
@@ -654,6 +654,41 @@ export class Verifier {
     // }
 
     // TODO(jwp): verify shielding/mints balance
+
+    const shieldBalance: BufferMap<bigint> = new BufferMap()
+    const mintBalance: BufferMap<bigint> = new BufferMap()
+
+    for (const event of result.events) {
+      // TODO(jwp): handle native asset as output balance (will require decrypting notes and comparing)
```

Comment:
I think we may need another description with public values for shielding the native asset -- both to get values to balance internally and to avoid decryption

Reply:
Yea maybe, although I don't think this is blocking us since we know that is possible. I think we should kick that down the road, wdyt?

Reply:
Yeah I think we can definitely defer that to later